src/docs/product/dashboards/custom-dashboards/index.mdx

```diff
@@ -89,15 +89,29 @@ transactions'.
 
 ## Data Set Selection
 
-Select between events and [issues](/product/issues/) data in a widget with the "Data Set" selector.
+Select between events, [issues](/product/issues/) and [release health](/product/releases/health/) data in a widget with the "Data Set" selector.
 
+### All Events
 Choosing "All Events" allows you to query and aggregate error and transaction events in the same way you would for a [Discover query](/product/discover-queries/). Some widget examples include:
 - Tracking performance of an endpoint
 - Throughput by country
 - Users most affected by errors
 
+### Issues
 Choosing "Issues" allows you to query issue properties, such as `is:unresolved` or event properties such as `error.handled:0`, and to sort by issue fields such as `First Seen`. When searching on event properties, the search will return any issue that has one or more events matching the supplied event filters. Choose this data set if you want to customize a list of issues on your dashboard. Some widget examples include:
 - Issues assigned to your team
 - Most frequently occurring issues
 
 The "Issues" data set is only available in [table](#table-results) visualization widgets.
+
+### Releases
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+Choosing "Releases" allows you to query properites specific to a release of your application, primarily session data. Choose this data set if you want to display healthy, errored and crashed sessions and crash rates for your releases or projects on your dashboard. Doesn't apply to World Map. Some widget examples include:
```

Comment:
I've made the wording edits here, but this will end up living on a new page entirely

```suggestion
Choosing "Releases" allows you to query properties specific to a release of your application, primarily session data. Choose this dataset if you want to display healthy, errored, and crashed sessions and crash rates for your releases or projects on your dashboard. This dataset isn't available in for [world map](#world-map) visualizations. Some example widgets using this dataset include:
```

Reply:
Addressed in this commit https://github.com/getsentry/sentry-docs/pull/5083/commits/9f2e2ce97d8c0235582dbc665bc16691d5c26d36#diff-483fd5b737378ca17b303465d6a2c5e14eb79f0d6c0456458a5ad2465b6eb710R88